### DISCUSSION TITLE
New-PnPTeamsTeam from existing group - template parameter not working

### DISCUSSION THREAD

**Discussion by stefwanders** [pnp/powershell#576] [Q&A]
I've been trying to create Teams from specific groups by following command:

New-PnPTeamsTeam -GroupId $GroupID -Template EDU_Class
Error code: New-PnPTeamsTeam : Parameter set cannot be resolved using the specified named parameters.

New-PnPTeamsTeam -GroupId $GroupID , works perfectly, but creates the standard Team. 

New-PnPTeamsTeam -DisplayName BS -MailNickName BS -Description BS -Visibility Private -Template EDU_Class , works perfectly as well.

Is it a bug that I cannot use the template command on an existing group?

**Comment by marcdayton** [comment]
Getting similar issue.  When trying to use New-PnPTeamsTeam -GroupId $GroupID.  I get the error:  New-PnPTeamsTeam : Parameter set cannot be resolved using the specified named parameters.
